freqtrade/data/dataprovider.py

```diff
@@ -44,36 +44,50 @@ def available_pairs(self) -> List[Tuple[str, str]]:
 
     def ohlcv(self, pair: str, ticker_interval: str = None, copy: bool = True) -> DataFrame:
         """
-        get ohlcv data for the given pair as DataFrame
-        Please check `available_pairs` to verify which pairs are currently cached.
+        Get ohlcv data for the given pair as DataFrame
+        Please check `self.available_pairs` to verify which pairs are currently cached.
         :param pair: pair to get the data for
-        :param ticker_interval: ticker_interval to get pair for
-        :param copy: copy dataframe before returning.
-                     Use false only for RO operations (where the dataframe is not modified)
+        :param ticker_interval: ticker interval to get data for
+        :param copy: copy dataframe before returning if True.
+                     Use False only for read-only operations (where the dataframe is not modified)
         """
         if self.runmode in (RunMode.DRY_RUN, RunMode.LIVE):
-            if ticker_interval:
-                pairtick = (pair, ticker_interval)
-            else:
-                pairtick = (pair, self._config['ticker_interval'])
+            pairtick = (pair, ticker_interval or self._config['ticker_interval'])
+            if pairtick in self.available_pairs:
```

Comment:
I think you can safely ommit this check, the same check is in the `_exchange.klines()` method too.

Reply:
Right, done.